windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,6 +17,18 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
 
 You must have a subscription to receive support for the Red Hat WMCO. Deploying Windows container workloads in production clusters is not supported without a subscription. If you do not have a subscription, you can use the community WMCO, a distribution that lacks official support. Request support through the link:http://access.redhat.com/[Red Hat Customer Portal].
 
+[id="wmco-2-0-2"]
+== Release notes for Red Hat Windows Machine Config Operator 2.0.2
+
+Issued:2021-07-XX
+
+The WMCO 2.0.2 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHSA-XXXX:YYYY[RHSA-XXXX:YYYY].
+
+[id="wmco-2-0-2-bug-fixes"]
+=== Bug fixes
+
+* Previously, when you created a Windows pod, the status would remain stuck in "ContainerCreating." This bug was due to the introduction of the Windows-unsupported `os.Chown()` method. Now, when you start a Windows pod, the status updates to "Running" once the Windows pod is active. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1975553[**BZ#1975553**])
```

Comment:
The issue with the Pod stuck in `ContainerCreating` shows up only in 4.8. The reason for backporting the fix is to allow for upgrades to go through smoothly. Scenario is a 4.7.z cluster with Windows Pods and is upgraded to 4.8.z and WMCO is not upgraded to 3.0.0 it will result in the Pods being stuck in `ContainerCreating`. This is because the `BoundServiceAccountTokenVolume` is enabled by default in 4.8 and that results in Projected Volumes being attached to every Pod. In addition `RunAsUser` is added to the `SecurityContext` of the Pod and this is the root cause that triggers the Projected Volume bug with `os.Chown()` being executed for a Windows Pod and results in a failure. 

TL;DR: Customers need to upgrade to WMCO 2.0.2 before upgrading their cluster to 4.8 if they have active Windows Pods.